tla/MCetcdraft.cfg

```diff
@@ -65,5 +63,4 @@ INVARIANTS
     LogMatchingInv
     QuorumLogInv
     MoreUpToDateCorrectInv
-    LeaderCompletenessInv
-    CommittedIsDurableInv
```

Comment:
Did you deliberately remove `CommittedIsDurableInv`?

Reply:
Yes, the invariable I added previously is invalid so I remove it.